src/Mapping/MappingException.php

```diff
@@ -329,6 +329,24 @@ public static function fileMappingDriversRequireConfiguredDirectoryPath(string|n
         );
     }
 
+    /**
+     * Returns an exception that indicates that discriminator entries used in a discriminator map
+     * does not exist in the backed enum provided by enumType option.
+     *
+     * @param array<int,int|string> $entries     The discriminator entries that could not be found.
+     * @param string                $owningClass The class that declares the discriminator map.
+     * @param string                $enumType    The enum that entries were checked against.
+     */
+    public static function invalidEntriesInDiscriminatorMap(array $entries, string $owningClass, string $enumType): self
+    {
+        return new self(sprintf(
+            "The entries %s in the discriminator map of class '%s' do not correspond to enum cases of '%s'.",
```

Comment:
You're wrapping the second `%s` with quotes, why not the first one as well? It might be empty.